### DISCUSSION TITLE
Reticulum Wiki

### DISCUSSION THREAD

**Discussion by Erethon** [markqvist/Reticulum#473] [Show and tell]
A couple of months ago Mark enabled the Wiki functionality of this repository. Seeing as it hasn't been used a lot, I want to invite everyone to start populating pages and information they think are interesting for this project but don't belong in the manual/documentation.

As an example first page I've created a list of ["third party" Reticulum related software](https://github.com/markqvist/Reticulum/wiki/Awesome-Reticulum).

**Comment by markqvist** [comment]
Thanks @Erethon! Good with a reminder for everyone :)

**Comment by cobraPA** [comment]
Nice... I have added links to the WIP iPhone areas.  I will get to doing some 'how to setup' and 'what is working' wiki info for the iPhone Beta.  Any preference to if that goes here or in the wiki's on my GitHub pages?
